src/analyze/analyze.c

```diff
@@ -1832,6 +2010,50 @@ static int do_security(int argc, char *argv[], void *userdata) {
         return analyze_security(bus, strv_skip(argv, 1), 0);
 }
 
+static int parse_arg_color_style(DotArgColorStyle color_or_style, const char *arg_dot_color_style) {
```

Comment:
please, let's leave the `arg_` prefix for the global variables that are parsed from the cmdline and other process parameters. For local variables and function params let's leave the prefix off, to not make things needlessly confusing

Reply:
done